ui/src/app/cohort-search/search-group-item/search-group-item.component.tsx

```diff
@@ -141,7 +140,7 @@ interface State {
   loading: boolean;
   paramListOpen: boolean;
   renaming: boolean;
-  timeout: Timeout;
+  timeout: NodeJS.Timeout;
```

Comment:
It was, yes. Typescript compiler couldn't figure out the type otherwise. I think that either Angular may have had a Timeout built in, or previously IntelliJ was looking at `window.setTimeout` or something. The error I was getting was `Type 'number' is not assignable to type 'Timeout'`.